apps/music/elements/music-search-results.js

```diff
@@ -5,157 +5,149 @@
 var proto = Object.create(HTMLElement.prototype);
 
 proto.createdCallback = function() {
-  var shadowRoot = this.createShadowRoot();
-  shadowRoot.innerHTML =
-`<style>
-  [data-icon]:before { /* Copied from /components/gaia-icons/gaia-icons.css */
-    font-family: "gaia-icons";
-    content: attr(data-icon);
-    display: inline-block;
-    font-weight: 500;
-    font-style: normal;
-    text-decoration: inherit;
-    text-transform: none;
-    text-rendering: optimizeLegibility;
-    font-size: 30px;
-    -webkit-font-smoothing: antialiased;
-  }
-  music-search-results {
-    opacity: 0;
-    visibility: hidden;
-    transition: opacity 0.2s linear, visibility 0s linear 0.2s;
-    pointer-events: none;
-  }
-  music-search-results[active] {
-    opacity: 1;
-    visibility: visible;
-    transition-delay: 0s, 0s;
-    pointer-events: auto;
-  }
-  #list {
-    background: var(--background);
-    position: relative;
-    width: 100%;
-    height: 100%;
-    z-index: 99999;
-  }
-  #list a {
-    color: var(--text-color);
-    text-decoration: none;
-    padding: 9px 0;
-    width: 100%;
-  }
-  #list li {
-    padding: 0;
-  }
-  #list h3,
-  #list p {
-    background: none !important;
-    padding-right: 60px;
-  }
-  #list img {
-    right: 0;
-    left: auto !important;
-    object-fit: cover;
-  }
+  this.innerHTML = `<style scoped>
+music-search-results {
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+}
+
+music-search-results.transitions-on {
+  transition:
+    opacity 0.2s linear,
+    visibility 0s linear 0.2s;
+}
+
+music-search-results[active] {
+  opacity: 1;
+  visibility: visible;
+  transition-delay: 0s, 0s;
+  pointer-events: auto;
+}
+
+#list {
+  background: var(--background);
+  position: relative;
+  width: 100%;
+  height: 100%;
+  z-index: 99999;
+}
+
+#list a {
+  color: var(--text-color);
+  text-decoration: none;
+  padding: 9px 0;
+  width: 100%;
+}
+
+#list li {
+  padding: 0;
+}
+
+#list h3,
+#list p {
+  background: none !important;
+  padding-right: 60px;
+}
+
+#list img {
+  right: 0;
+  left: auto !important;
+  object-fit: cover;
+}
 </style>
 <gaia-fast-list id="list">
   <template>
-    <li>
-      <a href="\${url}" data-file-path="\${name}" data-section="\${section}">
-        <div class="image"><img></div>
-        <h3>\${title}</h3>
-        <p>\${subtitle}</p>
-      </a>
-    </li>
+    <a href="\${url}" data-file-path="\${name}" data-section="\${section}">
+      <div class="image"><img></div>
+      <h3>\${title}</h3>
+      <p>\${subtitle}</p>
+    </a>
   </template>
 </gaia-fast-list>`;
 
-  var $id = shadowRoot.getElementById.bind(shadowRoot);
-
   this.els = {
-    list: $id('list')
+    list: this.querySelector('#list')
```

Comment:
`HTMLElement` doesn't have that method. Only `Document` and `DocumentFragment` IIRC.